tests/console/mariadb_srv.pm

```diff
@@ -35,6 +36,23 @@ sub run {
     systemctl "start $mariadb";
     systemctl "status $mariadb";
     assert_screen 'test-mysql_srv-1';
+
+    # Test multiple instance configuration
+    assert_script_run "touch /etc/mynode1.cnf";
+    assert_script_run "touch /etc/mynode2.cnf";
+    assert_script_run "curl " . data_url('console/mariadb/mynode1.cnf') . " -o /etc/mynode1.cnf";
+    assert_script_run "curl " . data_url('console/mariadb/mynode2.cnf') . " -o /etc/mynode2.cnf";
+    assert_script_run "mkdir -p /var/lib/mysql/node1";
+    assert_script_run "mkdir -p /var/lib/mysql/node2";
+    assert_script_run "chown mysql:root /var/lib/mysql/node1";
+    assert_script_run "chown mysql:root /var/lib/mysql/node2";
+
+    systemctl 'start mariadb@node1.service';
+    systemctl 'start mariadb@node2.service';
```

Comment:
Since you're starting two nodes, it would make a lot of sense to also bring down the nodes after the test is done, just make sure it's also called in the post_fail_hook just in case the test fails..